.pre-commit-config.yaml

```diff
@@ -82,7 +82,7 @@ repos:
         name: mypy-tests
         args: [--ignore-missing-imports]
         files: ^(packages/|docs|/conftest.py|src/tests)
-        exclude: (^packages/.*/setup.py|/src|^packages/aiohttp/aiohttp_patch.py$|^packages/zfpy/test_zfpy.py$)
+        exclude: (^packages/.*/setup.py|/src|^packages/aiohttp/aiohttp_patch.py$|^packages/zfpy/test_zfpy.py$|^packages/netcdf4/test_netcdf4.py$)
```

Comment:
I wonder if it's possible to exclude files by putting a pragma at the top instead of listing them here.

Reply:
Ah, there is: https://mypy.readthedocs.io/en/stable/common_issues.html#ignoring-a-whole-file, will implement this change, except for the glob patterns.

Reply:
Thanks!